libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/TextComposer.kt

```diff
@@ -395,6 +398,26 @@ private fun TextFormatting(
                 imageVector = ImageVector.vectorResource(VectorIcons.CodeBlock),
                 contentDescription = stringResource(CommonStrings.rich_text_editor_code_block)
             )
+
+            var linkDialogAction by remember { mutableStateOf<LinkAction?>(null) }
+
+            linkDialogAction?.let {
+                TextComposerLinkDialog(
+                    onDismissRequest = { linkDialogAction = null },
+                    onCreateLinkRequest = state::insertLink,
+                    onSaveLinkRequest = state::setLink,
+                    onRemoveLinkRequest = state::removeLink,
+                    linkAction = it,
+                )
+            }
+
+            FormattingOption(
```

Comment:
In the designs, the link action seems to be the last one (which is kind of weird IMO). Should this be moved there? I'd have expected it to be after the inline formatting ones, to be honest.

Reply:
@callumu what do you think? I have added the button immediately after code block as per the designs but the this doesn't take into account the quote button. I'd also agree with @jmartinesp that the link button would work well if positioned immediately after the inline formatting buttons (between strike-though and lists).

Reply:
I'm happy to move it there 👍

Can you communicate this change to iOS to keep consistency between platforms?

Reply:
cc @alfogrillo, I've added to https://github.com/vector-im/element-x-ios/issues/1702